src/Simulation/Simulators/SparseSimulator/build.ps1

```diff
@@ -6,12 +6,20 @@
 Write-Host "##[info]Build SparseSimulator"
 
 & (Join-Path $PSScriptRoot .. .. .. .. build set-env.ps1)
-$FailureCommands = 'Write-Host "##vso[task.logissue type=error;] Failed to build SparseSimulator. See errors below or above." ;  Pop-Location ; Exit 1'
+$FaiureLogging = 'Write-Host "##vso[task.logissue type=error;] Failed to build SparseSimulator. See errors below or above."'
+$FailureExitCommand = 'Exit 1'
+$FailureCommands = "$FaiureLogging ;  Pop-Location ; $FailureExitCommand"
 
 # BULD NATIVE PART
 
 # mkdir Native\build\(Debug|Release)
 $BuildDir = (Join-Path $PSScriptRoot "Native" "build" $Env:BUILD_CONFIGURATION)
+# if (Test-Path $BuildDir -PathType Leaf) {   # if file rather than dir
+#     Remove-Item -Path $BuildDir -Force -ErrorVariable $Errors
+#     if ($Errors.Count -gt 0) {    # if failed to remove the file
+#         Invoke-Expression "$FaiureLogging ; $FailureExitCommand"
+#     }
+# }
```

Comment:
This section was commented out. Should it be removed?

Reply:
I accidentally prematurely merged a few commits of my own. Reverting.